### PR TITLE
Update player version

### PR DESCRIPTION
**Pull request by HalfbyteHeroes**
This PR bumps the player version to 8.193.0, as the `PlayerEvent.RestoringContent` event introduced together with https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/119 was forward-ported to this player version.

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
         box-shadow: 0 0 30px rgba(0, 0, 0, 0.7);
       }
     </style>
-    <script src="//cdn.bitmovin.com/player/web/8.158.1/bitmovinplayer.js"></script>
+    <script src="//cdn.bitmovin.com/player/web/8.193.0/bitmovinplayer.js"></script>
```
Comment (wasp898) [inline example/index.html]: Nit: Not sure if this should just be 8 to be honest.

**Review by wasp898 — COMMENTED** [review]
Please update the `since` in PlayerEvent 😬

File: example/index.html (hunk 1 above)
Comment (HalfbyteHeroes) [reply example/index.html]: Yep, fair point - updated to latest in 255991e8

**Review by wasp898 — APPROVED** [review]
Danke 🙏

**Pull request merged**